migrations/0035_update_order_payment_methods.down.sql

```diff
@@ -0,0 +1,2 @@
+alter table order_items drop column payment_methods;
+alter table order_items add column payment_methods text[];
```

Comment:
was this just for development? ( since this is the same definition in migration 34? ) can we combine these migrations?